### PR TITLE
doc api reset password

### PR DESCRIPTION
**Pull request by at-mailuong**
## Description
doc api reset password
## Migrations
NO

### CODE REVIEW

**Comment by at-khadinh** [comment]
LGTM

File: docs/api-references/resetpass.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+## Send Mail Resetpassword Api
+
+### `POST` Send Mail Resetpassword
+```
+/api/password/email
```
Comment (at-quangtran) [inline docs/api-references/resetpass.md]: /api/password/reset

File: docs/api-references/resetpass.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+```
+Post email
+#### Request Headers
+| Key | Value | 
+|---|---|
+| Accept | application/json
+
+#### Query Param
```
Comment (at-quangtran) [inline docs/api-references/resetpass.md]: #### Request Body

File: docs/api-references/resetpass.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        ]
+    },
+    "code": 422,
+    "request": {
+        "email": "admin@"
+    }
+}
+```
+
+### `POST` Resetpassword
```
Comment (at-quangtran) [inline docs/api-references/resetpass.md]: PUT

File: docs/api-references/resetpass.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+* _Error_
+``` json
+{
+    "error": {
+        "message": "We can't find a user with that e-mail address.",
+        "request": {
+            "email": "admin@test.com"
+        }
+    },
+    "code": 422
```
Comment (at-quangtran) [inline docs/api-references/resetpass.md]: not found là 404

File: docs/api-references/resetpass.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    "error": {
+        "message": "We can't find a user with that e-mail address.",
+        "request": {
+            "email": "admin@test.com",
+            "token": "5ed78c1a0d612dd7442b66ba901deb04580609ac4e606220ddb99fe3c8026257",
+            "password": "123456",
+            "password_confirmation": "123456"
+        }
+    },
+    "code": 422
```
Comment (at-quangtran) [inline docs/api-references/resetpass.md]: not found là 404

**Pull request merged**